modules/datadict/README.md

```diff
@@ -0,0 +1,43 @@
+# Data Dictionary
+
+## Purpose
+
+The data dictionary module is used for viewing and modifying the
```

Comment:
Instead of modifying:  "customizing behavioural field descriptions"
You can't really modify the data dictionary otherwise...

Reply:
you can modify any descriptions including imaging ones, so I just added "descriptions" to the end of the sentence to make it clear that you can't edit other parts of the data dictionary (ie. data types..)